sky/backends/cloud_vm_ray_backend.py

```diff
@@ -231,10 +231,15 @@ def add_gang_scheduling_placement_group(
             'CPU': backend_utils.DEFAULT_TASK_CPU_DEMAND
         } for _ in range(num_nodes)]
 
+        ray_supported_accs = backend_utils.supported_ray_accs()
```

Comment:
Sorry could you explain why is `supported_ray_accs` needed?
For example, we can `sky launch gpunode --gpus M60 --cloud aws` and Ray will correctly recognize M60 as a gpu even it's not on the list right?

```
ubuntu@ip-172-31-82-9:~$ ray status
======== Autoscaler status: 2022-11-01 21:22:40.810182 ========
Node status
---------------------------------------------------------------
Healthy:
 1 ray.head.default
Pending:
 (no pending nodes)
Recent failures:
 (no failures)

Resources
---------------------------------------------------------------
Usage:
 0.0/4.0 CPU
 0.0/1.0 GPU
 0.0/1.0 M60
 0.0/1.0 accelerator_type:M60
 0.00/17.552 GiB memory
 0.00/8.776 GiB object_store_memory

Demands:
 (no resource demands)
```

Reply:
Thanks Weilin for the quick review! 

When `M60` or other GPUs are detected by Ray, it will naturally add a `GPU` to the placement group under the hood, as you see in your M60 logs.

I tested this with a 1080 GPU yesterday, and the GPU field did not show up. So, to be safe, some of this code handles the cases where the `GPU` field does not show up.

Reply:
Do we understand why 1080 can't not be detected by Ray? I was asking because the [list](https://github.com/skypilot-org/skypilot/pull/1356/files#diff-5d3b8c053073eef6241a9fc4439b8e97c680e73e6dbfe01a587eea2bf0000c4dR2058) here isn't the cause as `M60` is not in it but can be correctly identified.

I traced Ray's code a bit. Looks like how they identify #num_gpu is by the function `_autodetect_num_gpus ` below
https://github.com/ray-project/ray/blob/f39d323ed5916b67042407231f5b91851e8fa0b5/python/ray/_private/resource_spec.py#L268
It's by either `GPUtil.getGPUs()` which relies on `nvidia-smi` or `/proc/driver/nvidia/gpus/` which relies on nvidia-driver I believe.

Then as long as `nvidia-smi` or its driver works, 1080 GPU should be correctly detected. Could you share more?

Reply:
Great find! This is my family member's GPU and its fair to say `nvdia-smi` was not installed and could be why Ray failed to detect.